privacy.html

```diff
@@ -0,0 +1,141 @@
+---
+layout: default
+title: Datenschutzhinweise / Privacy Statement
+---
+
+<!-- Page Heading/Breadcrumbs -->
+<div class="row">
+    <div class="col-lg-12">
+        <h1 class="page-header">Datenschutzhinweise / Privacy Statement
+        </h1>
+        <ol class="breadcrumb">
+            <li><a href="index.html">Home</a>
+            </li>
+            <li class="active">Privacy Statement</li>
+        </ol>
+    </div>
+</div>
+<div class="row">
+  <section class="col-md-6">
+    <p>Mit den folgenden Datenschutzhinweisen erläutern wir, welche personenbezogene Daten wir über Sie erheben, wenn Sie uns auf unserer Webseite openHAB.org besuchen und dort ggf. den my.openHAB-Online-Dienst nutzen] und wie wir Ihre Daten verwenden.</p>
+
+    <p><strong>Datenschutzerklärung</strong></p>
+
+    <p>Die Nutzung unserer Webseite ist in der Regel ohne Angabe personenbezogener Daten möglich.</p>
+
+    <p>Wir weisen darauf hin, dass die Datenübertragung im Internet (z.B. bei der Kommunikation per E-Mail) Sicherheitslücken aufweisen kann. Ein lückenloser Schutz der Daten vor dem Zugriff durch Dritte ist nicht möglich.</p>
+
+    <p>Der Nutzung von im Rahmen der Impressumspflicht veröffentlichten Kontaktdaten durch Dritte zur Übersendung von nicht ausdrücklich angeforderter Werbung und Informationsmaterialien wird hiermit ausdrücklich widersprochen. Die Betreiber der Seiten behalten sich ausdrücklich rechtliche Schritte im Falle der unverlangten Zusendung von Werbeinformationen, etwa durch Spam-Mails, vor.</p>
+  </section>
+
+  <section class="col-md-6">
+    <p>With the following statements we want to inform you, which personal information we handle about you, if you are using our website, mobile applications or the my.openHAB online service.</p>

```

Comment:
@kaikreuzer I added ",mobile applications" to this line, can u translate to German?